test/with_dummyserver/test_socketlevel.py

```diff
@@ -781,6 +781,7 @@ def echo_socket_handler(listener):
                              b'Accept: */*',
                              b'',
                              b'',
+                             b'User-Agent: python-urllib3/dev'
```

Comment:
This testcase will fail on any tagged version. Need to update this test to pass with any setting of `__version__`.

Reply:
Good catch. Yeah, we need to get the version dynamically - since we're working outside urllib3, we should be able to do `from urllib3 import __version__ as package_version` and use that variable.